packages/react-select/src/Select.tsx

```diff
@@ -1624,9 +1721,12 @@ export default class Select<
       'aria-labelledby': this.props['aria-labelledby'],
       'aria-required': required,
       role: 'combobox',
+      'aria-activedescendant': this.isAppleDevice
```

Comment:
When server-side rendering this hydration throws an error
`Warning: Extra attributes from the server: aria-activedescendant`
Setting either both to empty sting `''` or `undefined` solves
`'aria-activedescendant': this.isAppleDevice ? '' : this.state.focusedOptionId || ''`
or
`'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || undefined`

Reply:
any update on the issue ?